credmark/cmf/model/context.py

```diff
@@ -370,3 +372,81 @@ def run_model(self,  # type: ignore
             ModelBaseError: other subclasses of ``ModelBaseError`` that should not be caught.
 
         """
+
+    class BlockNumber(int):
```

Comment:
Please move this back to types. Circular dependencies can be resolved.

Reply:
reverted